app/controllers/publications_controller.rb

```diff
@@ -133,20 +133,19 @@ def wrap_as_bibjson_collection(description, records, page = 1, per_page = 'all')
     # @param [Author] author
     # @return [String] contains csv report of an author's publications
     def generate_csv_report(author)
-      csv_str = CSV.generate do |csv|
+      CSV.generate do |csv|
         csv << %w(sul_pub_id sciencewire_id pubmed_id doi wos_id title journal year pages issn status_for_this_author created_at updated_at contributor_cap_profile_ids)
         author.publications.find_each do |pub|
-          journ = pub.pub_hash[:journal] ? pub.pub_hash[:journal] : { name: '' }
+          journ = pub.pub_hash[:journal] ? pub.pub_hash[:journal][:name] : ''
           contrib_prof_ids = pub.authors.pluck(:cap_profile_id).join(';')
```

Comment:
Maybe `pub.pub_hash.dig(:journal, :name) || ''` works here?
```ruby
[1] pry(main)> x = { journal: {} }
=> {:journal=>{}}
[2] pry(main)> x.dig(:journal, :name)
=> nil
[3] pry(main)> x = { journal: { name: 'name' } }
=> {:journal=>{:name=>"name"}}
[4] pry(main)> x.dig(:journal, :name)
=> "name"
```

Reply:
Yes, but that's not the point.  We're not worried about traversing the depths of the hash.  The point was to make `journ` a string instead of a structured object that only ever gets used to extract the string.

Reply:
Yes, but dig is better - consider when/if `pub.pub_hash[:journal] == {}` - the current code will assign `nil` to `journ` and that's not what you want here (and looks like it was a bug before too).